src/averageAges.js

```diff
@@ -37,7 +43,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter(person => person.sex === 'f'
+    && people.some(child => child.mother === person.name))
```

Comment:
According to the code style guideline, a particular chunk of code should look like this:
```
    ? people.filter(person => (
    person.sex === 'f' && people.some(child => child.mother === person.name)
    ))
```